packages/analytics/dashboard-renderer/src/components/BarChartRenderer.vue

```diff
@@ -3,7 +3,7 @@
   <BaseAnalyticsChartRenderer
     :chart-options="chartOptions"
     :context="context"
-    :editable="editable"
+    :editable="context.editable"
```

Comment:
You're already passing in `context` -- no need for the extra prop.  :)

Reply:
done